seqr/views/utils/orm_to_json_utils.py

```diff
@@ -711,20 +711,41 @@ def _process_result(result, locus_list):
         if include_genes:
             intervals = _get_json_for_models(interval_set.all())
             genome_versions = {interval['genomeVersion'] for interval in intervals}
+
+            if include_pagenes:
+                result.update({
+                    'items': [{
+                        'geneId': gene.gene_id,
+                        'pagene': _get_json_for_model(gene.palocuslistgene, user=user)
+                        if hasattr(gene, 'palocuslistgene') else None
+                    } for gene in gene_set.all()] + intervals,
+                })
+            else:
+                result.update({
+                    'items': [{'geneId': gene.gene_id} for gene in gene_set.all()] + intervals,
+                })
             result.update({
-                'items': [{'geneId': gene.gene_id} for gene in gene_set.all()] + intervals,
                 'intervalGenomeVersion': genome_versions.pop() if len(genome_versions) == 1 else None,
             })
+
         if include_project_count:
             result['numProjects'] = locus_list.num_projects
         result.update({
             'numEntries': gene_set.count() + interval_set.count(),
             'canEdit': user == locus_list.created_by,
         })
 
+        if hasattr(locus_list, 'palocuslist'):
+            pa_locus_list_json = _get_json_for_model(locus_list.palocuslist, user=user)
+            result.update({
+                'paLocusList': pa_locus_list_json,
+            })
+
     prefetch_related_objects(locus_lists, 'created_by')
     prefetch_related_objects(locus_lists, 'locuslistgene_set')
     prefetch_related_objects(locus_lists, 'locuslistinterval_set')
+    prefetch_related_objects(locus_lists, 'palocuslist')
```

Comment:
the prefetch should probably be conditional on `include_pagenes`

Reply:
Fixed.